src/notation/internal/notationactioncontroller.cpp

```diff
@@ -2173,10 +2189,24 @@ void NotationActionController::playSelectedElement(bool playChord)
     currentNotationScore()->setPlayNote(false);
 }
 
+bool NotationActionController::startNoteInputAllowed() const
+{
+    if (isEditingElement()) {
+        return false;
+    }
+
+    if (hasSelection() && !noteOrRestSelected()) {
+        return false;
+    }
+
+    const muse::ui::UiContext ctx = uiContextResolver()->currentUiContext();
+    return ctx == muse::ui::UiCtxProjectFocused && qApp->focusWindow();
```

Comment:
Or maybe use `QGuiApplication::applicationState() == Qt::ApplicationActive`?

Reply:
Yeah, that's better, fixed